### DISCUSSION TITLE
Undo/Redo missing from key configs

### DISCUSSION THREAD

**Discussion by WangleLine** [ppy/osu#30977] [Ideas]
Hi. I have recently started using the lazer editor, and I quickly noticed that you cannot seem to change the keybinds for undoing/redoing in the editor. I use a german keyboard layout, for context, and osu! is making me use a different key combination for undoing than what I'm used to.

I searched the list of issues and found [this comment mentioning something related, in passing](https://github.com/ppy/osu/issues/11143#issuecomment-1532115197), but I think even without non-US keyboard layouts in mind, it would be tremendously useful if we could change the keybinds for these actions.

**Comment by peppy** [comment]
I don't think we'd offer rebinding of system-level keys. But we definitely want to fix the non-QWERTY layout failures.
